### PR TITLE
Task/APPS-3300 — homepage revisions

### PR DESCRIPTION
**Pull request by giuliacampana**

### CODE REVIEW

File: src/containers/home/HomePage.js [hunk 1]
```diff
@@ -90,69 +88,66 @@ const HomePage = () => (
...
           <Tile>
             <Typography maxWidth={{ lg: 528 }} variant="body2">
-              We’re creating new ways for academic researchers to collect, analyze and advance their research with
-              Chronicle - a modern  Study Management tool that expands research capabilities. Chronicle enables
-              researchers and primary investigators to better understand participant behavior and unlocks new insights
-              with mobile functionality, ecological momentary assessment features, and customizable study management
-              workflows.
+              Enroll participants, collect consent, track protocol compliance, compensate participants, and
+              integrate passive passive sensing, experience sampling, surveys and other data sources for
+              analysis in one secure platform.
```
Comment (anzioka) [inline src/containers/home/HomePage.js:99]: `passive` seems to be duplicated

Comment (giuliacampana) [reply src/containers/home/HomePage.js:99]: oh thank you! very helpful

**Review by anzioka — APPROVED** [review]

File: src/assets/images/home/index.js [hunk 2]
```diff
@@ -18,8 +19,14 @@ const LaptopImg = styled.img.attrs({
   src: LaptopPNG,
 })``;
 
+const PhoneImg = styled.img.attrs({
+  alt: 'laptop',
```
Comment (solomontang) [inline src/assets/images/home/index.js:23]: shouldn't this alt be `phone` or something?

File: src/assets/images/home/index.js [hunk 3]
```diff
@@ -6,6 +6,7 @@ import styled from 'styled-components';
 
 import DataSecurityAndCompliancePNG from './data-security-and-compliance.png';
 import LaptopPNG from './laptop.png';
+import PhonePNG from './phone.png';
```
Comment (solomontang) [inline src/assets/images/home/index.js:9]: has this been compressed already? .3MB seems pretty heavy. https://tinypng.com/ or https://squoosh.app/ are great online tools to do this very quickly

File: src/assets/logos/index.js [hunk 4]
```diff
@@ -4,20 +4,20 @@
...
 
-import BCMLogoPNG from './research-logos/Baylor-College-of-Medicine.png';
-import BYULogoPNG from './research-logos/BYU.png';
+import BCMLogoPNG from './research-logos/baylor-college-of-medicine.svg';
+import BYULogoPNG from './research-logos/BYU.svg';
 import CFRLogoPNG from './42cfrp2.png';
 import CJISLogoPNG from './cjis.png';
 import FERPALogoPNG from './ferpa.png';
-import GeorgetownLogoPNG from './research-logos/Georgetown-University.png';
+import GeorgetownLogoPNG from './research-logos/Georgetown.png';
```
Comment (solomontang) [inline src/assets/logos/index.js:12]: how did this png get 7x bigger?

File: src/assets/images/home/index.js (hunk 2 above)
Comment (giuliacampana) [reply src/assets/images/home/index.js:23]: oops yes, thank you!

File: src/assets/images/home/index.js (hunk 3 above)
Comment (giuliacampana) [reply src/assets/images/home/index.js:9]: i actually have compressed it once already... was originally 1.8MB

File: src/assets/logos/index.js (hunk 4 above)
Comment (giuliacampana) [reply src/assets/logos/index.js:12]: ah it's a different png actually—different logo.

**Review by geekbeast — APPROVED** [review]

**Pull request merged**